src/config/config.json

```diff
@@ -4,5 +4,21 @@
   "mongodb": {
     "url": "mongodb://localhost/babajka",
     "options": {}
+  },
+  "auth": {
+    "saltRounds": 8
+  },
+  "trustProxy": 0,
+  "session": {
+    "secret": "todo: replace on production",
```

Comment:
@UladBohdan we should discuss how and where we should store this `secret` option